src/com/google/enterprise/adaptor/filenet/FileNetObjectFactory.java

```diff
@@ -30,6 +30,10 @@
  * Factory for producing instances various FileNet Objects.
  */
 class FileNetObjectFactory implements ObjectFactory {
+  @Override
+  public FileNetAdaptor.Traverser getTraverser(ConfigOptions options) {
```

Comment:
I had that originally, but it made sense to me to wait and see what it looked like when we needed it.